doc/api/tls.md

```diff
@@ -1023,6 +1023,9 @@ being issued by trusted CA (`options.ca`).
 <!-- YAML
 added: v0.11.3
 changes:
+  - version: REPLACEME
+    pr-url: REPLACEME
```

Comment:
`https://github.com/nodejs/node/pull/25517`. Just pointing it out because I know it's easy to forget.

Reply:
Yes will do just wasn't sure what id to use before submitting.